source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.ylabel('Surface Albedo')
</pre><p> </p><p>I like adding [-] just to denote that it is unitless and be consistent with some of the other plots where you include the square bracket and units if it's temperature. I don't feel very strongly though, so feel free to ignore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>